nixos/modules/services/security/endlessh-go.nix

```
@@ -0,0 +1,96 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.endlessh-go;
+in
+{
+  options.services.endless-go = {
+    enable = mkEnableOption "Whether to enable the endlessh-go service.";
```

```suggestion
    enable = mkEnableOption "the endlessh-go service";
```
the rest is added automatically